src/dependencies.js

```diff
@@ -0,0 +1,75 @@
+'use strict'
+
+const dependencies = require('electron-installer-common/src/dependencies')
```

Comment:
We need to export this function on the `-common` module. I also think that the `index.js` on the `-common` module needs some cleaning. Probably move the definitions of functions to another file and keep `index.js` solely for importing/exporting of needed functions.